internal/config/projectconfig/init.go

```diff
@@ -59,3 +44,33 @@ func Init(dir string, projectName string, projectContext *ZeroProjectConfig) {
 		exit.Fatal(fmt.Sprintf("Failed to create config file %s", constants.ZeroProjectYml))
 	}
 }
+
+func GetProjectFileContent(projectConfig ZeroProjectConfig) string {
+	var tpl bytes.Buffer
+	tmpl := template.New("sa")
```

Comment:
You can chain these two lines. `tmpl, err := template.New("projectConfig").tmp.Parse(zeroProjectConfigTemplate)`